packages/block-library/src/video/edit.native.js

```diff
@@ -64,7 +64,7 @@ class VideoEdit extends React.Component {
 
 	componentDidMount() {
 		const { attributes } = this.props;
-		if ( attributes.id && attributes.url && ! isURL( attributes.src ) ) {
```

Comment:
I remember like `attributes.url` was not becoming undefined when we close/open a post during the upload. but now it is, so it is now useless and preventing the block from calling `mediaUploadSync`. maybe sth changed in the infrastructure that is causing this but I think removing this altogether should be fine.